workers/data_refinery_workers/processors/smasher.py

```diff
@@ -393,14 +393,14 @@ def _smash(job_context: Dict) -> Dict:
 
                     # Explicitly title this dataframe
                     try:
-                        data.columns = [computed_file.samples.all()[0].title]
+                        # Unfortuantely, we can't use this as `title` can cause a collision
+                        #data.columns = [computed_file.samples.all()[0].title]
+                        # So we use this, which also helps us support the case of missing SampleComputedFileAssociation
+                        data.columns = [computed_file.filename]
                     except ValueError:
```

Comment:
Is this except block still necessary? I think the only exception that could be raised here would be an AttributeError.

Reply:
Changed to base exception, which I think is necessary for two-channel samples